CactbotOverlay/FFXIVProcess.cs

```diff
@@ -512,6 +573,16 @@ public class EntityData {
             data.debug_job += string.Format("{0:x2}", job_bytes[i]);
           }
         }
+
+        for (int i = 0; i < data.statusEffects.Length; i++) {
+          var currentBase = kEntityStructureOffsetCharacterDetails + kEntityStructureOffsetStatusEffects + i * StatusEffect.structSize;
+
+          var id = BitConverter.ToUInt16(bytes, currentBase + StatusEffect.offsetId);
+          var stackCount = bytes[currentBase + StatusEffect.offsetStackCount];
+          var duration = BitConverter.ToSingle(bytes, currentBase + StatusEffect.offsetDuration);
+          var casterId = BitConverter.ToUInt32(bytes, currentBase + StatusEffect.offsetCasterId);
```

Comment:
From a usability perspective, do you think `casterId` matters past "is this my effect"? I'm mostly just wondering if this should be a bool.